tests/testthat/test-shinytest2-tm_a_pca.R

```diff
@@ -0,0 +1,164 @@
+app_driver_tm_a_pca <- function() {
+  # Dataset only used once
+  data <- within(teal.data::teal_data(), {
+    require(nestcolor)
+
+    USArrests <- USArrests # nolint: object_name.
+  })
+  teal.data::datanames(data) <- "USArrests"
+
+
+  init_teal_app_driver(
+    data = data,
+    modules = tm_a_pca(
+      dat = teal.transform::data_extract_spec(
+        dataname = "USArrests",
+        select = teal.transform::select_spec(
+          choices = teal.transform::variable_choices(
+            data = data[["USArrests"]],
+            c("Murder", "Assault", "UrbanPop", "Rape")
+          ),
+          selected = c("Murder", "Assault"),
+          multiple = TRUE
+        )
+      )
+    )
+  )
+}
+
+testthat::test_that("e2e - tm_a_pca: Data selection (data_extract) changes eigenvector table", {
```

Comment:
Test doesn't check if the table changed but if "UrbanPop" word is missing in the table content. Please adjust the test or the description